README.md

```diff
@@ -70,6 +70,23 @@ void main() {
 }
 ```
 
+A `Dashbook` instance can also support localizations (i18n) and default language
+
+```dart
+final dashbook = Dashbook(
+    localizationsDelegates: [
+      AppLocalizations.delegate, // Custom Localization
```

Comment:
Can we have some better explanations on what is this? Or even just a link to some Flutter documentation. I don't think this is totally clear what or how the localization here works